src/Helper/Parser.php

```diff
@@ -178,6 +178,22 @@ public static function parseDateMDYReadable(string $date): ?\DateTimeImmutable
         return null;
     }
 
+    /**
+     * @param string $dateTime
+     * @return \DateTimeImmutable|null
+     */
+    public static function parseDateTimePST(string $dateTime) : ?\DateTimeImmutable
+    {
+        try{
+            $malTimeZone = new \DateTimeZone('America/Los_Angeles'); //
```

Comment:
`America/Los_Angeles` TZ is used instead of `PST` since PHP seems to return -08 offset for PST even now in daylight savings time.